### PR TITLE
Add quantized ops to pybindings

### PR DESCRIPTION
**Pull request by lucylq**

### CODE REVIEW

File: examples/models/llama2/runner/generation.py [hunk 1]
```diff
@@ -19,9 +19,20 @@
...
 )
+
+from executorch.extension.pybindings import portable_lib  # noqa
+# Note: import this after portable_lib
+from executorch.extension.llm.custom_ops import sdpa_with_kv_cache  # noqa
+
 from executorch.extension.pybindings.portable_lib import _load_for_executorch
 
 
+from executorch.kernels import quantized
```
Comment (larryliu0820) [inline examples/models/llama2/runner/generation.py:30]: import executorch.kernels.quantized  # noqa[F401] 'executorch.kernels.quantized' imported but unused

File: .ci/scripts/test_pybind.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import torch
```
Comment (larryliu0820) [inline .ci/scripts/test_pybind.py:1]: For this test, can we do the following things:

* Add it into the other test modules https://github.com/pytorch/executorch/tree/main/extension/pybindings/test
* Use `unittest` framework
* Add a test for exporting an nn.Module with quantized ops to executorch, read it back using pybind and run it.

Comment (lucylq) [reply .ci/scripts/test_pybind.py:1]: Ah, thanks

Comment (lucylq) [reply .ci/scripts/test_pybind.py:1]: Referred to https://github.com/pytorch/executorch/blob/main/exir/tests/test_quant_fusion_pass.py#L40 for quant test.

**Review by larryliu0820 — APPROVED** [review]

File: extension/pybindings/test/test_pybindings.py [hunk 3]
```diff
@@ -13,6 +13,15 @@
     )
 
     kernel_mode = "portable"
+    # Load portable lib.
```
Comment (larryliu0820) [inline extension/pybindings/test/test_pybindings.py:16]: Would be good to mention that we have to load portable_lib first

**Pull request merged**